docs/clients: Add missing sensor responses (#71)
Filiprogrammer/AlphabotSZUCar: docs/wifi_ble_communication_protocol.md

| Field Name | Field Type | Notes                                   |
|------------|------------|-----------------------------------------|
| Distance   | uint16     | The distance to anchor 0 in centimeters |
| Distance   | uint16     | The distance to anchor 1 in centimeters |
| Distance   | uint16     | The distance to anchor 2 in centimeters |

### Wheel speed response (Packet ID: 0x09)

| Field Name | Field Type | Notes                          |
|------------|------------|--------------------------------|
| Speed      | int8       | The speed of the wheels in m/s |

### Gyroscope response (Packet ID: 0x0A)

| Field Name | Field Type | Notes                                    |
|------------|------------|------------------------------------------|
| X-axis     | int16      | The speed of the x-axis in degree/s * 10 |
| Y-axis     | int16      | The speed of the y-axis in degree/s * 10 |
| Z-axis     | int16      | The speed of the z-axis in degree/s * 10 |

### Accelerometer response (Packet ID: 0x0B)

| Field Name | Field Type | Notes                                         |
|------------|------------|-----------------------------------------------|

> | X-axis     | int16      | The acceleration of the x-axis in m/s² * 1000 |
> | Y-axis     | int16      | The acceleration of the y-axis in m/s² * 1000 |
> | Z-axis     | int16      | The acceleration of the z-axis in m/s² * 1000 |

Reviewer: ```suggestion
| X-axis     | int16      | The acceleration of the x-axis in m/s&#xB2; * 1000 |
| Y-axis     | int16      | The acceleration of the y-axis in m/s&#xB2; * 1000 |
| Z-axis     | int16      | The acceleration of the z-axis in m/s&#xB2; * 1000 |
```

### Magnetometer response (Packet ID: 0x0C)

| Field Name | Field Type | Notes                                              |
|------------|------------|----------------------------------------------------|

> | X-axis     | int16      | The magnetic flux density of the x-axis in µT * 10 |
> | Y-axis     | int16      | The magnetic flux density of the y-axis in µT * 10 |
> | Z-axis     | int16      | The magnetic flux density of the z-axis in µT * 10 |

Reviewer: ```suggestion
| X-axis     | int16      | The magnetic flux density of the x-axis in &#xB5;T * 10 |
| Y-axis     | int16      | The magnetic flux density of the y-axis in &#xB5;T * 10 |
| Z-axis     | int16      | The magnetic flux density of the z-axis in &#xB5;T * 10 |
```

| Position X | int16      | The x coordinate in centimetres of anchor 2                 |
| Position Y | int16      | The y coordinate in centimetres of anchor 2                 |

### BLE_CHAR_ANCHORS_DISTANCES_WHEEL_SPEED (UUID: 254492a2-9324-469b-b1e2-4d4590972c35)

| Field Name | Field Type | Notes                                   |
|------------|------------|-----------------------------------------|
| Distance   | uint16     | The distance to anchor 0 in centimetres |
| Distance   | uint16     | The distance to anchor 1 in centimetres |
| Distance   | uint16     | The distance to anchor 2 in centimetres |

> | Speed      | int8       | The speed of the wheels in m/s          |

Reviewer: I don't like it... it feels weird

Reviewer: After all, the sensor characteristic still has an unused sensor type.

Author: yeah i done it

> ### BLE_CHAR_IMU (UUID:93758afa-ce6f-4670-9562-ce92bda84d49)

Reviewer: ```suggestion
### BLE_CHAR_IMU (UUID: 93758afa-ce6f-4670-9562-ce92bda84d49)
```

| Field Name | Field Type | Notes                                                                                                                                   |
|------------|------------|-----------------------------------------------------------------------------------------------------------------------------------------|
| Type       | int8       | Indicates whether the packet is a gyroscope-package (value 0), an accelerometer-package (value 1), or an magnetometer-package (value 2) |
| x-Axis     | int16      | The value of the x-Axis                                                                                                                 |
| y-Axis     | int16      | The value of the y-Axis                                                                                                                 |

> | z-Axis     | int16      | The value of the z-Axis                                                                                                                 |

Reviewer: Wouldn't it be better to send all of the IMU sensor data at once?

Author: no because maybe else you can not differentiate when one sensor is not available (alphabot with only 2 sensors for example)
